enhancements/network/primary-user-defined-networks-for-virtualization-workloads.md

```diff
@@ -0,0 +1,588 @@
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type IPAMClaimList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []IPAMClaim `json:"items"`
+}
+```
+
+This CRD is owned by the k8snetworkplumbingwg; updating it requires quorum from
+the community.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+The management cluster should have no reason to use multiple networks, unless
+it for security reasons it makes sense to use native network isolation over
+network policy.
+
+It makes more sense that multiple primary networks will be used in the hosted
+cluster in order to provide tenants with better isolation from each other,
+without the need for network policy.
+There should be no hypershift platform-specific considerations with this
+feature.
+
+#### Standalone Clusters
+
+Full support.
+
+#### Single-node Deployments or MicroShift
+
+OpenShift Virtualization is not supported on `MicroShift` clusters, hence,
+we do not plan to integrate with UDN on these type of clusters right now.
+
+It is tracked in this [jira issue](https://issues.redhat.com/browse/RFE-5289).
+
+Still, the passt integration is known for imposing some additional overhead
+which might not be suited for MicroShift. This could be mitigated by using
+bridge binding instead, which is more conservative in its resource consumption.
+
+### Implementation Details/Notes/Constraints
+
+N/A
+
+### Risks and Mitigations
+
+Some packets will definitely be dropped during live-migration of the VM; while
+we want to have the migration as seamless as possible, just preserving the
+5 tuple of the existing connections is far from optimal.
+
+If the tests show that the traffic disruption takes too long, we should
+prioritize the
+[seamless live-migration epic](https://issues.redhat.com/browse/CNV-27147)
+which makes use of the multi requested-chassis OVN feature, in which a single
+logical switch port can be bound to multiple chassis (and have traffic
+forwarded to them).
+
+We have requested the core OVN team to ensure this feature is in fact usable on
+InterConnect clusters (which OpenShift is). This request is tracked in this FDP
+[issue](https://issues.redhat.com/browse/FDP-770).
+
+### Drawbacks
+
+N/A
+
+## Test Plan
+
+- E2E upstream and downstream jobs covering supported features across multiple
+networks.
+- E2E tests ensuring VM live-migration preserving established TCP connections. 
+- E2E tests covering reachability to external networks (VM egress over UDN
+networks for Layer2 topologies). 
+- E2E tests covering a VM exposed as a `LoadBalancer` service is reachable from
+the outside world. 
+- Scale testing to determine limits and impact of multiple user-defined
+networks.
+
+## Graduation Criteria
+
+### Dev Preview -> Tech Preview
+
+There will be no dev or tech preview for this feature.
+
+### Tech Preview -> GA
+
+Targeting GA in OCP version 4.17.
```

Comment:
```suggestion
Targeting GA in OCP version 4.17.z.
```

Reading just 4.17 I think 4.17.0, which A) won't happen and B) becomes confusing while reading about binding released in 4.18.

Reply:
Done.